src/backend/cuda/nvrtc/cache.cpp

```diff
@@ -149,6 +157,107 @@ void Kernel::getScalar(T &out, const char *name) {
 template void Kernel::setScalar<int>(const char *, int);
 template void Kernel::getScalar<int>(int &, const char *);
 
+#if !defined(OS_WIN)
+string getHomeDirectory() {
+    string home = getEnvVar("XDG_CONFIG_HOME");
```

Comment:
Shouldn't this be `XDG_CACHE_HOME` for kernel cache ?

Reply:
Didn't know about it! Will change